src/main/java/pages/MainPage.java

```diff
@@ -0,0 +1,78 @@
+package pages;
+
+import java.util.List;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import utils.WaitActions;
+
+public class MainPage {
+
+    @FindBy(className = "benefit-icon")
+    public List<WebElement> listOfImages;
+
+    @FindBy(className = "benefit-txt")
+    public List<WebElement> listOfTextUnderImages;
+
+    @FindBy(id = "frame")
+    public WebElement frame;
+
+    @FindBy(id = "frame-button")
+    public WebElement frameButton;
+
+    @FindBy(xpath = "//*[@class='sidebar-menu left']/li")
+    public List<WebElement> leftMenu;
+
```

Comment:
Let's move headerMenu variable just before or after your locators for better readability.